addons/dark-www/scrollbar.css

```diff
@@ -1,16 +1,32 @@
 body:not(.sa-body-editor)::-webkit-scrollbar,
 body:not(.sa-body-editor) ::-webkit-scrollbar {
-  background: #333;
+  width: 16px;
+  height: 16px;
+  background: transparent;
+}
+body:not(.sa-body-editor)::-webkit-scrollbar {
+  background: #202020;
 }
 body:not(.sa-body-editor)::-webkit-scrollbar-thumb,
 body:not(.sa-body-editor) ::-webkit-scrollbar-thumb {
```

Comment:
there are the same lines, or does having a space do something?

Reply:
`body:not(.sa-body-editor) ::-webkit-scrollbar-thumb` is equivalent to `body:not(.sa-body-editor) *::-webkit-scrollbar-thumb`: the scrollbar of any element inside the body, but not the body itself.